lms/static/sass/multicourse/_course_about.scss

```diff
@@ -26,7 +31,11 @@
 
       margin: 0 auto;
       max-width: map-get($container-max-widths, xl);
-      min-width: 760px;
+
+      @media screen and (min-width: $bp-screen-md) {
+        min-width: $bp-screen-md;
+      }
```

Comment:
If you use the bootstrap stuff from above, you can also use container-max-widths, as in map-get($container-max-widths, md) instead of 960px;